frontend/.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
Should root be set on this one, since we have another higher in the tree? Also we should remove js and css 4 space from the top level editorconfig

Reply:
Well.. I'm not 100% sure about root.. however, in reference to to removing.. I don't think so.. because the inner will just overwrite the outter.. I don't think it will hurt anything to be in both